### ISSUE TITLE
fix exponential transport loss factor

### ISSUE THREAD

**Issue by jacob-mannhardt** [ZEN-universe/ZEN-garden#653]
If transport_loss_factor_exponential = 0, then flow_transport_loss = flow_transport, thus, the entire flow is a loss.
Fix and reactivate test_1d
